website/project/views/node.py

```diff
@@ -1008,6 +1008,66 @@ def get_children(auth, node, **kwargs):
     return _render_nodes(nodes, auth)
 
 
+def node_child_tree(user, node_ids):
```

Comment:
This function and the view that uses it still makes me a bit nervous. Are you 100% sure that this function wouldn't reveal any information that a user shouldn't be able to see (e.g. private project viewed by a user who is a read-write on a parent, etc.)? On the flip-side, are you sure that it does show all the information that `user` has permission to see (e.g. an admin on a project should be able to view the title of all the project's children).

There need to be tests to verify these.

The advantage of using the same code as the wiki or logs is code sharing--the code would only need to be tested in one place. I think Brian was suggesting using the same code from NodeLogs for gathering the tree structure, then building the Treebeard structure from that. Is this a feasible approach, @billyhunt ?